rediscluster/utils.py

```diff
@@ -203,3 +203,53 @@ def parse_slots(s):
         nodes.append(node)
 
     return nodes
+
+
+def parse_pubsub_channels(command, res, **options):
```

Comment:
Also `parse_cluster_nodes` is defined in `utils.py`, which of course is only used by `CLUSTER NODES` and `CLUSTER SLAVES`.

I put them in there mostly as I assumes it was the place you'd want them, given that's where all the other parsers are. Same with redis-py

```python
from .utils import (
    bool_ok,
    string_keys_to_dict,
    dict_merge,
    blocked_command,
    merge_result,
    first_key,
    clusterdown_wrapper,
    parse_cluster_slots,
    parse_cluster_nodes,
    parse_pubsub_channels,
    parse_pubsub_numsub,
    parse_pubsub_numpat,
)
```
It also keeps this nice and neat at the top of `client.py`